unstructured/partition/csv.py

```diff
@@ -29,18 +30,18 @@
 @add_chunking_strategy
 def partition_csv(
     filename: Optional[str] = None,
-    file: Optional[Union[IO[bytes], SpooledTemporaryFile]] = None,
```

Comment:
just curious, but why were you able to get rid of the SpooledTemporaryFile type?

Reply:
`IO[bytes]` is implemented by`SpooledTemporaryFile`, i.e. `SpooledTemporaryFile` _is a_ `IO[bytes]`. So specifying `SpooledTemporaryFile` along with `IO[bytes]` would be redundant in the same way that `Fruit | Oranges` would be. Oranges are a fruit, so are already covered as a permissible type by `Fruit`.